pyopensprinkler/station.py

```diff
@@ -48,13 +83,63 @@ def seconds_remaining(self):
         """Retrieve seconds remaining"""
         return self._get_status_variable(1)
 
+    @property
+    def start_time(self):
+        """Retrieve start time"""
+        return self._get_status_variable(2)
+
+    @property
+    def max_name_length(self):
+        return self._controller._state["stations"]["maxlen"]
+
+    @property
+    def masop(self):
+        return self._bit_check("masop")
+
+    @property
+    def masop2(self):
+        return self._bit_check("masop2")
+
+    @property
+    def ignore_rain(self):
+        return self._bit_check("ignore_rain")
+
+    @property
+    def ignore_sn1(self):
```

Comment:
`ignore_rain_sensor_1`

Reply:
I did `sensor_1_ignored` as the sensor may or may not be a rain sensor.